src/Toolkit/Toolkit/UI/Controls/PopupViewer/TextPopupElementView.Maui.cs

```diff
@@ -49,17 +55,422 @@ private static object BuildDefaultTemplate()
+                    bool isOrdered = node.Token?.Name == "ol";
+                    var listGrid = new Grid { Margin = new Thickness(0, 0, 0, 16) };
+                    listGrid.ColumnDefinitions.Add(new ColumnDefinition { Width = GridLength.Auto }); // bullets
+                    listGrid.ColumnDefinitions.Add(new ColumnDefinition { Width = GridLength.Star }); // contents
+
+                    var childItems = node.Children.Where(n => n.Type == MarkupType.ListItem).ToList(); // ignore a misplaced non-list-item node
+
+                    for (int row = 0; row < childItems.Count; row++)
+                    {
+                        listGrid.RowDefinitions.Add(new RowDefinition { Height = GridLength.Auto });
+                        var markerText = isOrdered ? $"{row + 1}." : "\u2022";
+                        listGrid.Add(new Label { Text = markerText, HorizontalTextAlignment = TextAlignment.End, Margin = new Thickness(5, 0) }, 0, row);
+                        var item = CreateBlock(childItems[row]);
+                        listGrid.Add(item, 1, row);
+                    }
+                    return listGrid;
+
+                case MarkupType.Block:
+                case MarkupType.ListItem:
+                case MarkupType.TableCell:
+                    bool isPara = node.Token?.Name == "p";
+                    View view;
+                    if (HasAnyBlocks(node))
+                    {
+                        var container = new StackLayout();
+                        if (isPara)
+                            container.Margin = ParagraphMargin;
+                        if (node.BackColor.HasValue)
+                            container.BackgroundColor = ConvertColor(node.BackColor.Value);
+
+                        var blocks = VisitChildren(node);
+                        foreach (var block in blocks)
+                        {
+                            container.Children.Add(block);
+                        }
+                        view = container;
+                    }
+                    else
+                    {
+                        var label = CreateFormattedText(node.Children);
+                        if (isPara)
+                            label.Margin = ParagraphMargin;
+                        ApplyStyle(label, node);
+                        view = label;
+                    }
+                    if (node.Type == MarkupType.TableCell)
+                        return VerticallyAlignTableCell(node, view);
+                    return view;
+
+                case MarkupType.Divider:
+                    return new BoxView { HeightRequest = 1, Color = Colors.Gray }; // TODO: Do we need to set a color?
+
+                case MarkupType.Table:
+                    return ConvertTableToGrid(node);
+
+                case MarkupType.Image:
+                    var imageElement = new Image();
+                    if (Uri.TryCreate(node.Content, UriKind.Absolute, out var imgUri))
```

Comment:
Note that his can also begin with `data:image/`. See https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/1e8ad8a249796bea954778bcb5360465b1ae430e/src/Toolkit/Toolkit/UI/Controls/PopupViewer/PopupMediaView.cs#L89-L92

Reply:
Looks like I missed this from both MAUI and WPF text element views.  Fixed!

![2023-09-11_144705](https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/assets/587809/7412ae2e-2080-41bb-8fc4-9d17fbd37a87)